src/test/kotlin/org/openbroker/model/StraigtLineAprTest.kt

```diff
@@ -0,0 +1,52 @@
+package org.openbroker.model
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.openbroker.loancosts.StraightLine
+
+class StraightLineAprTest {
+    @Disabled
```

Comment:
Why is these tests disabled?

Reply:
I never completed APR computations for StraightLine. They are (also) a bit tricky to implement, and right now we have no obvious need for it, so I decided to not complete that part of the code, given how much other things we have to do right now. So the tests are disabled because they will currently fail, but we are okay with that for the moment.

Reply:
I guess I could exclude the straight line bit and add it to another PR that we can keep open.